File: fatawa-client-Full-update-with-new-design/src/app/infrastructure/models/project/UserModel.ts
export interface UserModel {
    userId: number;
    userName: string;
    email: string;
    phoneNumber: string;
    isActive: boolean;
    dateOfBirth: Date;
    roleId: number;
    roleName: string;
    parentId: number;
    fullName: string;
}
export class UserTree {
    userId: number;
    userName: string;
    email: string;
    parentId: number;
}

File: fatawa-server-master/TeamWork.Core/Models/Security/UserModel.cs
using System;
using System.Collections.Generic;

namespace TeamWork.Core.Models.Security
{
    public class UserModel
    {
        public int UserId { get; set; }
        public string UserName { get; set; }
        public string Email { get; set; }
        public string PhoneNumber { get; set; }
        public bool IsActive { get; set; }
        public DateTime DateOfBirth { get; set; }
        public int LanguageId { get; set; }
        public int RoleId { get; set; }
        public string RoleName { get; set; }
        public string FullName { get; set; }

    }
}
